main/views.py

```diff
@@ -169,34 +169,33 @@ def github_code_to_member(code, ohmember):
             'client_id': settings.GITHUB_CLIENT_ID,
             'client_secret': settings.GITHUB_CLIENT_SECRET
         }
-        req = requests.post(
-            'https://github.com/login/oauth/access_token'.format(
-                settings.OPENHUMANS_OH_BASE_URL),
-            data=data
-        )
+        # Add headers telling Github's API that we want a JSON response (instead of plaintext)
+        headers = {'Accept': 'application/json'}
+        # Get the access_token from the code
+        req = requests.post('https://github.com/login/oauth/access_token',
+                            data=data, headers=headers)
         data = req.json()
         print(data)
+        # Now that we have a token, let's get the users "profile" back with their token:
+        auth_string = 'token {}'.format(data['access_token'])
+        token_header = {'Authorization': auth_string}
+        user_data_r = requests.get('https://api.github.com/user', headers=token_header)
+        user_data = user_data_r.json()
```

Comment:
Because the token response only contains a token, type, and scope as such `{"access_token":"abc123token", "scope":"repo,gist", "token_type":"bearer"}` we'll need to make one additional API call (using the newly granted token) to get that user's profile so that we can store their `github_id`. This request is to `https://api.github.com/user` and returns the github user info - [see an example response object here](https://developer.github.com/v3/users/#response). We can then use the `user_data['id']` to get that user's unique Github ID.